From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Shelley Vohr <shelley.vohr@gmail.com>
Date: Wed, 14 Jul 2021 21:09:22 -0700
Subject: fix: use new V8 error message property access format

Refs changes in https://chromium-review.googlesource.com/c/v8/v8/+/3013935

This patch can be removed when Node.js updates to a version of V8
which contains the above CL.

diff --git a/test/parallel/test-fs-read.js b/test/parallel/test-fs-read.js
index 5c9e59795fc39f871748d5971453e3b6e2238b6b..16cae5486671e3ee60be77d986b25805649798c7 100644
--- a/test/parallel/test-fs-read.js
+++ b/test/parallel/test-fs-read.js
@@ -80,7 +80,10 @@ assert.throws(
 
 assert.throws(
   () => fs.read(fd, { buffer: null }, common.mustNotCall()),
-  /TypeError: Cannot read property 'byteLength' of null/,
+  {
+    name: 'TypeError',
+    message: 'Cannot read properties of null (reading \'byteLength\')'
+  },
   'throws when options.buffer is null'
 );
 
